Source/ServiceCall.cpp

```diff
@@ -106,13 +107,54 @@ static int handleGetParameters(void** params, int len){
       *value = settings.output_offset;
     }else if(strncmp(SYSEX_CONFIGURATION_OUTPUT_SCALAR, p, 2) == 0){
       *value = settings.output_scalar;
+    }else if(strncmp(SYSEX_CONFIGURATION_RESOURCE_COUNT, p, 2) == 0){
```

Comment:
why would a patch need to know how many resources there are?

Reply:
This is leftover from being able to index all resources and access them by ID. Without ID based indexing it won't be useful, so I'll be removing it (and delete operation)